docs/compute-element/hosted-ce.md

```diff
@@ -0,0 +1,139 @@
+OSG Hosted CE
+=============
+
+
+Introduction
+------------
+
+An OSG Compute Element (CE) is the entry point for jobs coming from the OSG;
+it handles authorization and delegation of jobs to your existing campus HPC/HTC cluster.
+
+Many sites set up their compute element locally.
+As an alternative, OSG offers a hosted CE option
+wherein the OSG team will host and operate the HTCondor compute element,
+and configure it for the science communities that you choose to support.
+
```

Comment:
Let's add a sentence here saying that the Hosted-CE can support thousands of concurrent job submissions

Reply:
updated